packages/components/src/components/icon/common/ic-puzzle.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><path fill="var(--fill-color1)" d="M13.648 6.904c-.053 0-.106.001-.158.005V4.234c0-.95-.774-1.724-1.724-1.724H9.09A2.354 2.354 0 006.745 0a2.354 2.354 0 00-2.346 2.51H1.724C.774 2.51 0 3.284 0 4.234V7.48a.469.469 0 00.625.442A1.416 1.416 0 012.51 9.255 1.416 1.416 0 01.625 10.59a.468.468 0 00-.625.44v3.246C0 15.226.773 16 1.724 16H4.97a.469.469 0 00.434-.646 1.416 1.416 0 011.34-1.864 1.416 1.416 0 011.334 1.885.469.469 0 00.442.625h3.246c.95 0 1.724-.773 1.724-1.724v-2.675A2.354 2.354 0 0016 9.255a2.354 2.354 0 00-2.352-2.351z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><path fill="#FFF" fill-rule="evenodd" d="M12.56 14.5H8.99a.318.318 0 01-.183-.577c.376-.267.598-.7.598-1.161 0-.782-.63-1.422-1.408-1.422-.774 0-1.408.637-1.408 1.422 0 .461.222.894.598 1.16.11.08.16.225.117.357a.315.315 0 01-.3.221H3.44a.316.316 0 01-.314-.317V11.31a2.011 2.011 0 01-1.09.324A2.047 2.047 0 010 9.578c0-1.134.915-2.055 2.035-2.055.392 0 .771.116 1.091.324V4.973c0-.174.14-.316.314-.316h2.845a2.073 2.073 0 01-.32-1.102C5.965 2.42 6.88 1.5 8 1.5c1.124 0 2.035.924 2.035 2.055 0 .396-.114.779-.32 1.102h2.845c.173 0 .314.142.314.316v2.874a2.011 2.011 0 011.09-.324c1.125 0 2.036.924 2.036 2.055a2.048 2.048 0 01-2.035 2.056c-.392 0-.771-.116-1.091-.324v2.873c0 .175-.14.317-.314.317"/></svg>
```

Comment:
We should use CSS vars here so dark theme won't break.

Reply:
ahh right. I forget about it. Thanks 👍